### PR TITLE
Added Javascript Challenges

### PR DESCRIPTION
**Pull request by HarshCasper**
### Have you read the Contributing Guidelines on Pull Requests?

Yes

### Description

Added Javascript Challenges from `Question_1` to `Question_10`.

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` according to the changes I have made.

### CODE REVIEW

**Review by AkashM398 — APPROVED** [review]

File: JavaScript/Question_02/question_02.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+// It is returning [ ["flutes", 4] ]
+console.log(filteredArray([ ["trumpets", 2], ["flutes", 4], ["saxophones", 2] ], 2));
+
+// It is returning [ ["amy", "beth", "sam"] ]
+console.log(filteredArray([ ["amy", "beth", "sam"], ["dave", "sean", "peter"] ], "peter"));
+
+// It should return [ ]
+console.log(filteredArray([[3, 2, 3], [1, 6, 3], [3, 13, 26], [19, 3, 9]], 3));
```
Comment (KamalDGRT) [inline JavaScript/Question_02/question_02.js]: Kindly add a new line character at the end of this file.

File: JavaScript/Question_03/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Problem Description
+
+Given a Number `n` implement Recursion to create a COuntdown Timer.
```
Comment (KamalDGRT) [inline JavaScript/Question_03/README.md]: Change "COuntdown" to "Countdown"

File: JavaScript/Question_03/question_03.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        myArray.unshift(n);
+        return myArray;
+    }
+}
+
+// It should return [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
+console.log(countdown(10));
+
+// It should return [5, 4, 3, 2, 1]
+console.log(countdown(5));
```
Comment (KamalDGRT) [inline JavaScript/Question_03/question_03.js]: Add a new line character to represent End-Of-File.

**Review by KamalDGRT — CHANGES_REQUESTED** [review]
### Kindly do the following changes :

- Change the spelling in README.md of question_03.js file.
- Add a new line to denote the EOF in the following files : 
  - question_02.js
  - question_03.js
  - question_04.js
  - question_05.js
  - question_07.js
  - question_08.js
  - question_09.js
  - question_10.js
- Remove the extra blank line in these files. (Line number 5)
  - README.md of question 7
  - README.md of question 9

**Comment by HarshCasper** [comment]
Made Changes to the Files. I am merging them now.

**Pull request merged**